src/storage/src/source/mysql/snapshot.rs

```diff
@@ -284,7 +308,7 @@ pub(crate) fn render<G: Scope<Timestamp = TransactionId>>(
                     }
 
                     // TODO(roshan): Figure out how to add a test-case that ensures we do release this lock
-                    trace!(%id, "timely-{worker_id} releasing table locks");
+                    // trace!(%id, "timely-{worker_id} releasing table locks");
```

Comment:
Is this meant to be a comment?

Reply:
Nope, got left behind after doing the formatting. nice spot!